AK Zeus/app/src/main/java/ayds/zeus/songinfo/home/model/entities/SpotifySong.kt

```diff
@@ -1,11 +1,13 @@
 package ayds.zeus.songinfo.home.model.entities
 
+
 interface Song {
     val id: String
     val songName: String
     val artistName: String
     val albumName: String
     val releaseDate: String
+    val releaseDatePrecision:String
```

Comment:
Mal espaciado. Tiene que estar separada de los : la palabra String

```suggestion
    val releaseDatePrecision: String
```